lib/cn/omf.h

```diff
@@ -191,9 +191,9 @@ struct kblock_hdr_omf {
 
     /* easily accessible copies of min and max keys */
     uint32_t kbh_min_koff;
-    uint32_t kbh_min_klen;
+    uint16_t kbh_min_klen;
```

Comment:
Nabeel did this as part of his vblock PR to start using uint16_t for klen since HSE_KVS_KEY_LEN_MAX fits in a u16.